source/docs/advanced/theme/wysiwyg-platform.md

```diff
@@ -141,3 +154,48 @@ However, in most cases, you will need to fetch data to display all the needed in
     }
     ```
 1.  Use the fetched data in the `data` props in your final widget component (the `./path/to/ProductPreview.js` mentioned above)
+
+## `PrismicWysiwyg`
```

Comment:
**issue:** I feel having the documentation here mixes concerns and will make things more difficult to maintain over time.\
It is duplicated here and on the Prismic page.

**suggestion:** keep the `PrismicWysiwyg` here with a short paragraph explaining what it is. But redirect the reader to https://deploy-preview-394--heuristic-almeida-1a1f35.netlify.app/docs/prismic/embed-fields.html#Wysiwyg-Embed-Fields for further details